internal/handlers/tigris/tigrisdb/tigrisdb.go

```diff
@@ -20,23 +20,26 @@ import (
 
 	"github.com/tigrisdata/tigris-client-go/config"
 	"github.com/tigrisdata/tigris-client-go/driver"
+	"go.uber.org/zap"
 
 	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 )
 
 // TigrisDB represents a Tigris database connection.
 type TigrisDB struct {
 	Driver driver.Driver
+	L      *zap.Logger
```

Comment:
@AlekSi does this look ok'ish to you? or should it be through `opts` or something?

Reply:
Good enough for now